src/App.tsx

```diff
@@ -15,36 +23,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortBy {
+  'default',
+  'alpabet',
+  'length',
+}
+
+function getGoods(goodsFromServList: string[]): GoodsList {
+  return goodsFromServList.map((str, index) => ({ id: index, val: str }));
+}
+
+function prepGoods(goods: GoodsList, oper: SortBy, revOp: boolean): GoodsList {
```

Comment:
The function name `prepGoods` does not start with a verb and is not very descriptive. Consider renaming it to `prepareGoods` or `sortAndReverseGoods` to better reflect its purpose and follow the [CODE STYLE] guideline.